tcms/telemetry/api.py

```diff
@@ -180,8 +179,7 @@ def test_case_health(query=None):
     all_test_executions = TestExecution.objects.filter(**query)
 
     test_executions = _get_count_for(all_test_executions)
-    failed_test_executions = _get_count_for(all_test_executions.filter(
-        status__name__in=TestExecutionStatus.failure_status_names))
+    failed_test_executions = _get_count_for(all_test_executions.filter(weight < 0))
```

Comment:
your filter statement is wrong. I'm wondering how it didn't fail during testing ?

Reply:
this code still hasn't been manually tested (therefore the WIP PR). i need to update a few more places, where the old methods are called and then I can continue with manually testing everything